.github/workflows/prepare_release.yaml

```
@@ -0,0 +1,53 @@
+
+  release:
+    name: Prepare a new release
+    needs: [wait_for_checks]
+    if: |
+      always() &&
+      (needs.wait_for_checks.result == 'success' || needs.wait_for_checks.result == 'skipped')
+    uses: EnviroDIY/workflows/.github/workflows/prepare_release.yaml@main
+    secrets:
+      PLATFORMIO_AUTH_TOKEN: ${{ secrets.PLATFORMIO_AUTH_TOKEN }}
+    with:
+      include_dependencies: ${{ ( true ) || (github.event_name == 'workflow_dispatch' && github.event.inputs.publish == 'true') }}
```

actionlint (1.7.12)

[error] property "publish" is not defined in object type {include_dependencies: string}

(expression)